src/devices/Nmea0183/tests/Nmea0183.Tests.csproj

```diff
@@ -11,5 +11,8 @@
   <ItemGroup>
     <ProjectReference Include="..\Nmea0183.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Shouldly" Version="4.0.3" />
```

Comment:
Also consider moving the version of the package to versions.props.

Reply:
Will do when we find that this library is helpful elsewhere.